hooks/lib/provisioning_wizard.rb

```diff
@@ -294,8 +294,10 @@ def validate_network
   def validate_own_gateway
     if !(valid_ip?(@own_gateway))
       'Host Gateway - Invalid IP address'
-    elsif (IPAddr.new(from)..IPAddr.new(to))===IPAddr.new(own_gateway)
-      'DHCP range is Invalid - DHCP range includes the Host Gateway IP address'
+    elsif !from.nil? && !from.empty? && !to.nil? && !to.nil?
```

Comment:
I believe it should be
`elsif !from.nil? && !from.empty? && !to.nil? && !to.empty?`